src/components/Tree/Tree.stories.tsx

```diff
@@ -5,208 +5,44 @@ import { Meta, Story } from "@storybook/react";
+import { IconFile, IconFolder } from "@foundation/Icon";
 
-    return (
-        <Flyout
-            isOpen={open}
-            onOpenChange={(isOpen) => setOpen(isOpen)}
-            legacyFooter={false}
-            hug={false}
-            trigger={
-                <Button
-                    onClick={onClick}
-                    style={ButtonStyle.Secondary}
-                    icon={<IconActions size={IconSize.Size12} />}
-                    size={ButtonSize.Small}
-                />
-            }
-        >
-            <ActionMenu
-                menuBlocks={[
-                    {
-                        id: "dropdownBlock",
-                        ariaLabel: "Dropdown block",
-                        menuItems: [
-                            {
-                                id: "dropdown",
-                                size: MenuItemContentSize.Small,
-                                title: "Dropdown",
-                                onClick: () => console.log("dropdown"),
-                            },
-                        ],
-                    },
-                    {
-                        id: "actionsBlock",
-                        ariaLabel: "Item Actions",
-                        menuItems: [
-                            {
-                                id: "rename",
-                                size: MenuItemContentSize.Small,
-                                title: "Rename",
-                                onClick: () => console.log("rename"),
-                            },
-                            {
-                                id: "duplicate",
-                                size: MenuItemContentSize.Small,
-                                title: "Duplicate",
-                                onClick: () => console.log("duplicate"),
-                            },
-                            {
-                                id: "unpublish",
-                                size: MenuItemContentSize.Small,
-                                title: "Unpublish",
-                                onClick: () => console.log("unpublish"),
-                            },
-                        ],
-                    },
-                    {
-                        id: "deleteBlock",
-                        ariaLabel: "Delete Block",
-                        menuItems: [
-                            {
-                                id: "delete",
-                                size: MenuItemContentSize.Small,
-                                title: "Delete",
-                                onClick: () => console.log("delete"),
-                            },
-                        ],
-                    },
-                ]}
-            />
-        </Flyout>
-    );
+interface TreeListItem extends TreeNodeProps {
+    name: string;
+    parentId: TreeNodeProps["id"] | null;
+    sort: number | null;
```

Comment:
NullableNumber

Reply:
Nullable* types are not in arcade ATM. That kind of change should have a separate commit (it's not related to changes done in this PR). You can check #641 , it's all done there.